### ISSUE TITLE
您好，无法启动homo程序

### ISSUE THREAD

**Issue by hatleon** [countstarlight/homo#10]
（１）使用第一种方法，我执行这个命令cp conf/example_app.ini conf/app.ini，并没有改变app.ini的内容，运行./run.sh -a报错，** (homo-webview:13): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-aECIMHN3Zu: 拒绝连接
Gtk-Message: Failed to load module "canberra-gtk-module"；

（２）第二种，根据　https://homo.codist.me/docs/install/
这个网址进行配置，运行到make gen出现import cycle not allowed
package github.com/jteeuwen/go-bindata，好像是相互包依赖问题；

（３）谢谢。

**Comment by vufa** [comment]
使用docker方式启动时，docker容器会连接宿主机的图形服务来显示图形界面，这个出问题只能慢慢调试。
make gen是用go-bindata工具嵌入资源文件到源码的过程，可以手动安装这个工具再执行make gen
现在正在重构homo，使homo仅作为服务端，不再依赖gtk等图形界面服务
